### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by chrystalquek**
-Features introduction (convert note box to text)
-for commands that affect dashboard, include a notebox before the examples to state so (i.e. add/edit/mark done/mark undone/delete/clear deliverables and meetings…) [reference DG use case]
-move all list tip boxes to find command above and update the contents accordingly
-FAQ
-Command summary
-Glossary

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -74,27 +74,14 @@ Here is a sequence of example commands you can try:<br>
 
 ## Features
```
Comment (gabztcr) [inline docs/UserGuide.md:75]: Missing the table and command breakdown example (think we want to model after the UG example we saw ytd). Can refer to the pdf shared in the group (I also asked Clara to share the link to this model UG there) 

Specifically, for features:

-brief explanation on our features
-"Listed below are the common components of a command"
-(convert the notes box to table below)

-table
>>component (command word, prefix, parameter)
>>description

-Example
-"add t/TITLE by/DEADLINE m/MILESTONE [c/CONTACTS] [d/DESCRIPTION]"
-breakdown the example
-<Note box>: square brackets mean optional prefix-parameter pairs

File: docs/UserGuide.md [hunk 2]
```diff
@@ -212,6 +205,12 @@ Parameter      | Description                   | Example
 
 </div>
 
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:** The dashboard’s Schedule will be updated accordingly.
```
Comment (gabztcr) [inline docs/UserGuide.md:210]: I think this should belong in the existing notebox above, not in a separate one.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -364,12 +393,19 @@ Parameter      | Description                   | Example
 ---------------|-------------------------------|--------------------------------
 `INDEX`        | The index number of the meeting you wish to edit in your displayed meeting list. <br><br>Detail:<br>It must be a positive integer. |`1`: Represents the 1st meeting in your list
 
+<div markdown="block" class="alert alert-warning">
+
+**:warning: Warning:** You cannot edit a meeting to have the same `TITLE`, `FROM` and `TO` as an existing meeting.
```
Comment (gabztcr) [inline docs/UserGuide.md:398]: I don't remember that we agreed to include this warning though. I don't think the user needs to know this as he/she won't go out of the way to try out this constraint. Should we delete all of these warnings to do with sameness (here and elsewhere)? I recall we didn't want any warnings in the commands (not even for delete, clear).

File: docs/UserGuide.md [hunk 4]
```diff
@@ -587,72 +629,82 @@ Format: `clear`
...
 
-**Q**: How do I start using Productiv?<br>
-**A**: You can refer to our [Quick Start Guide](#quick-start).
+**Q**: Is Productiv available on both Windows and Mac?<br>
+**A**: Absolutely! Just ensure that you have Java `11` installed on your computer and it is your default Java version.
+
+**Q**: How many deliverables, meetings and contacts can I store in Productiv?<br>
+**A**: Productiv can support up to 1000 deliverables, 1000 meetings and 1000 contacts.
+
+**Q**: Can I transfer my data to another device?<br>
```
Comment (gabztcr) [inline docs/UserGuide.md:638]: I honestly feel that this FAQ seems abit too complicated to explain to users and is kinda prone to errors and messy instructions.
1. what does "device runs Productiv" mean? do we need to refer to quick start guide again?
2. is it 'copy over' or 'overwrite' ? and would they understand what these mean and the difference?
3. is it "./data" or just "data"

I personally prefer to take out this FAQ to minimise the communication breakdown. 4 FAQ left should be fine (they are explained well and concisely too). What do you think?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -587,72 +629,82 @@ Format: `clear`
...
-6  | Stakeholder                    | An external party involved with the product.                                                   |
+Term                | Description
+--------------------------------|------------------------------------------------------------------------------------------------|
+Command Line Interface (CLI)   | A text-based user interface (UI) used to view and manage computer files.                       |
+Graphical User Interface (GUI) | A system of interactive visual components for computer software.                               |                         |
+Milestone                      | A stage in the software development process associated with a particular group of deliverables. It is a non-negative integer, or a period-separated string of non-negative integers. |
+Mode                           | The state of the application that affects how each command will be executed. The app can be in dashboard, deliverable, meeting or contact mode.                                                  |
```
Comment (gabztcr) [inline docs/UserGuide.md:710]: Able to include OCP too? i.e. "Overall Completion Percentage (OCP) | A donut chart showing the product’s completion status, found on the left panel of the dashboard mode."

modified from DG.

File: docs/UserGuide.md (hunk 3 above)
Comment (chrystalquek) [reply docs/UserGuide.md:398]: yea thats what i remember also, im thinking whether we should delete all the warnings?

File: docs/UserGuide.md (hunk 4 above)
Comment (chrystalquek) [reply docs/UserGuide.md:638]: yep, this particular FAQ needs some step by step instructions, which shouldn't be the point of the FAQ.
I feel can remove.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -74,13 +74,32 @@ Here is a sequence of example commands you can try:<br>
 
 ## Features
 
-**Notes about the command format:**<br>
+Excited about what Productiv can do for you? Here are the 4 main features of Productiv!
```
Comment (gabztcr) [inline docs/UserGuide.md:77]: I'll shift these modes into my intro section in my commit but thank u

File: docs/UserGuide.md [hunk 7]
```diff
@@ -74,13 +74,32 @@ Here is a sequence of example commands you can try:<br>
...
+1. [Contact](#contact) - Helps you organise your developer or stakeholder contacts so that you can reference them easily.
 
-* Words in upper case are the parameters to be supplied by you for their respective fields.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter for the name field `n`, which can be used as `add n/Jason`.
 
-* Field-parameter pairs in square brackets are optional.<br>
-  e.g `n/NAME [p/PHONE]` can be used as `n/Jason p/98890112` or as `n/Jason`.
+**Listed below are the common components of a command:**<br>
+
+Component of Command | Description |
```
Comment (gabztcr) [inline docs/UserGuide.md:86]: Just 'Component' should suffice

File: docs/UserGuide.md [hunk 8]
```diff
@@ -74,13 +74,32 @@ Here is a sequence of example commands you can try:<br>
...
+Parameter |	The words following each prefix. It represents the value of the corresponding field. |
+
+For example, here's the breakdown of the command `add t/TITLE by/DEADLINE m/MILESTONE [c/CONTACTS] [d/DESCRIPTION]`:
+* Command word - `add`
+* Prefixes - `t/`, `by/`, `m/`, `c/`, `d/`
+* Parameters - `TITLE`, `DEADLINE`, `MILESTONE`, `CONTACTS`, `DESCRIPTION`
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Note:** Field-parameter pairs in square brackets are optional.
```
Comment (gabztcr) [inline docs/UserGuide.md:99]: Can consider to change to "Prefix-parameter pairs in square brackets are optional, i.e. `[c/CONTACTS]` and `[d/DESCRIPTION]` need not be present in the command." ?

**Review by gabztcr — APPROVED** [review]
LGTM less minor changes

**Pull request merged**